### DISCUSSION TITLE
Error when building KENS project with gcc

### DISCUSSION THREAD

**Discussion by systil3** [ANLAB-KAIST/KENSv3#138] [Project Q&A 1. Application Layer (Socket)] [answered]
When I build my project in gcc 6.3.0 mingw32 with wsl, this error occurs:

![ewfwfw](https://github.com/ANLAB-KAIST/KENSv3/assets/87977576/9d531ab3-57ce-43f2-bdd0-faac78027105)
Does anyone have the solution?

**Comment by Pusnow** [comment, marked as answer]
Ming**w32** is only for a Windows native environment not WSL.
Please, do your project using WSL shell or follow following instructions (working with WSL + VScode):

* https://learn.microsoft.com/en-us/windows/wsl/tutorials/wsl-vscode [[Korean]](https://learn.microsoft.com/ko-kr/windows/wsl/tutorials/wsl-vscode)
